### PR TITLE
add s02e01 and initial evaluation normal themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Thematic document: https://docs.google.com/spreadsheets/d/1bWqhcVDH7AI3FFgVAz5TgrRw3AmQTR1jp4SAHoVZRno/edit#gid=2043781094

### CODE REVIEW

File: code_schemes/dadaab_community_views_on_girls_education.json [hunk 1]
```diff
@@ -3,6 +3,46 @@
   "Name": "DADAAB Community Views On Girls Education",
   "Version": "0.0.0.2",
   "Codes": [
+    {
+      "CodeID": "code-dfc56c39",
+      "CodeType": "Normal",
+      "DisplayText": "yes",
+      "NumericValue": 1,
+      "StringValue": "yes",
```
Comment (as2388) [inline code_schemes/dadaab_community_views_on_girls_education.json]: Should this be "yes, other" given that there are other options for yes?

File: code_schemes/kakuma_community_views_on_girls_education.json [hunk 2]
```diff
@@ -3,6 +3,70 @@
   "Name": "KAKUMA Community Views On Girls Education",
   "Version": "0.0.0.2",
   "Codes": [
+    {
+      "CodeID": "code-a3d58611",
+      "CodeType": "Normal",
+      "DisplayText": "yes",
+      "NumericValue": 0,
+      "StringValue": "yes",
```
Comment (as2388) [inline code_schemes/kakuma_community_views_on_girls_education.json]: ditto re yes, other

**Review by as2388 — APPROVED** [review]

File: code_schemes/dadaab_community_views_on_girls_education.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/dadaab_community_views_on_girls_education.json]: This is for standalone yes/no responses. Happy to rename it to yes, no reason.

Comment (as2388) [reply code_schemes/dadaab_community_views_on_girls_education.json]: I think that's better (yes, no reason), and is what we used to do to solve this ambiguity when we saw it in the past.

Comment (IsaackMwenda) [reply code_schemes/dadaab_community_views_on_girls_education.json]: updated with no_reason suffix

**Pull request merged**